### ISSUE TITLE
Groups and Items height changes on horizontal scroll even though stack:false is used

### ISSUE THREAD

**Issue by vikasraj789** [almende/vis#3256]
Vis Timeline:
Version: 4.20
Browser: Chrome 56

Issue:- 
I have a requirement of keeping the items and groups height constant on scroll. I thought using stack:false option would solve this. I have tried solving it using css but couldn't. Please let me know if you need more info.

**Comment by bradh** [comment]
We need a reproduction scenario - please create one using jsbin / jsfiddle and link to that. Are you concerned that the size of a group box changes depending on what is displayed? I don't get the part about item height constant though - you need to explain this more.

**Comment by vikasraj789** [comment]
@bradh Yes, I am concerned about the size of group box. Please check the below link:
[http://visjs.org/examples/timeline/other/horizontalScroll.html](url)
On horizontal scroll, group box size changes if there are no items rendered for that group. I need to achieve constant(same min and max height) height to each group, so that even on scroll group size doesn't change. Below is the options object used for timeline.

```json
visOptions: {
        editable: {
            add: false,
            remove: false,
            updateGroup: true,
            updateTime: true
        },
        selectable: true,
        itemsAlwaysDraggable: true,
        showCurrentTime: true,
        end: moment('2017-07-12').set({ 'hour': 28, 'minute': 0, 'second': 0, 'millisecond': 0 }),
        start: moment('2017-07-12').set({ 'hour': 6, 'minute': 0, 'second': 0, 'millisecond': 0 }),
        format: {
            minorLabels: {
                hour: "ha"
            }
        },
        max: moment().add(2, 'M'),
        min: moment().subtract(2, 'M'),
        zoomable: false,
        horizontalScroll: true,
        verticalScroll: true,
        stack: false,
        orientation: {
            axis: 'top'
        },
        template: function (item, element) {
            if (!item) { return }
            ReactDOM.unmountComponentAtNode(element);
            return ReactDOM.render(<item.content item={item} />, element);
        },
        groupTemplate: function (group, element) {
            if (!group) { return }
            ReactDOM.unmountComponentAtNode(element);
            return ReactDOM.render(<group.content group={group} />, element);
        }
    }
```

**Comment by bradh** [comment]
So there is no issue with item height?

**Comment by vikasraj789** [comment]
@bradh No, the issue is with group height.

**Comment by vikasraj789** [comment]
@bradh waiting for the resolution.

**Comment by bradh** [comment]
@vikasraj789 I am busy. Sorry.

**Comment by vikasraj789** [comment]
@bradh ok.. When can I expect the resolution? Because this issue is blocking my delivery dates.

**Comment by bradh** [comment]
There is no schedule on my side. Sorry, but I have to do the paid work first. I can only repeat my suggestion that you provide the reproducer, and hope that someone else has time.

**Comment by vikasraj789** [comment]
@bradh Will try to do that ASAP.

**Comment by yotamberk** [comment]
@vikasraj789 i will remind you that all of use maintainers do not get paid for this work... Expecting a resolution is not fair or nice... If you want this to be resolved, you are more than welcome to contribute. Maybe your help will then get this issue to be prioritized... (You wash my hand, I wash yours...)

**Comment by vikasraj789** [comment]
@bradh @yotamberk  Sorry for the urgency created. Will do try to contribute to the project.

**Comment by hugosilva00** [comment]
@vikasraj789 Could you find a solution to the problem? I have the same requirement
